### ISSUE TITLE
Redis explained with RAID

### ISSUE THREAD

**Issue by colin-higgins** [SpykeBytes/PersonalWebsite#54]
Things to start with:
Usually you want a few a master and a few slaves. Slaves are used for the reads pretty much, master is used for the writes IIRC.
Make sure you set the threads available reasonably per instance.

Redis in cluster mode is composed of multiple shards. Each shard has a primary node and some number of replica nodes. What this is configuring is how many replicas you have per cluster (for fast reads from the replicas as well as intra-cluster data retention), and the number of clusters (and multi-az by default) provide multiple geographically distinct clusters in the case of single-az failures.

Basically think raid0=shard and raid1=replica. For your scale you shouldn’t need shards.
